app/views/customers/show.html.erb

```diff
@@ -0,0 +1,13 @@
+<h2>Customer Name: <%= @customer.name %></h2>
+<% @customer.items.each do |item| %>
+<p> Item Name: <%= item.name %> </p>
+<p> Item Price: <%= item.price %> </p>
+<p> Items Store: <%= item.supermarket.name %> </p>
+<% end %> 
+
+<%= form_with(url: "/customers/#{@customer.id}", local: true, method: :patch, id: 'add-item-form') do |f| %>
```

Comment:
Don't forget to disable Turbo! Not a problem for an IC but just reminding so that you don't forget in the future and see some weird behavior